src/components/form-sections/HasNextEnrollment/index.js

```diff
@@ -0,0 +1,40 @@
+import React, { useEffect, useState } from 'react';
+import { isEmpty } from 'lodash';
+import { getNextEnrollment } from '../../../lib/services';
+
+const HasNextEnrollment = ({ enrollmentId }) => {
+  const [nextEnrollment, setNextEnrollment] = useState(false);
+
+  useEffect(() => {
+    async function fetchNextEnrollment() {
+      if (!enrollmentId) return setNextEnrollment(false);
+
+      const fetchedNextEnrollment = await getNextEnrollment(enrollmentId);
+
+      setNextEnrollment(fetchedNextEnrollment);
+    }
+
+    fetchNextEnrollment();
+  }, [enrollmentId]);
+
+  if (isEmpty(nextEnrollment)) {
+    console.log('caca');
+    return null;
+  }
+
+  const nextEnrollementId = nextEnrollment[0].id;
```

Comment:
while there is a control on uniqueness of copied request ([see :eye:](https://github.com/betagouv/signup-back/blob/dd5776cf7436406dd839ff9376633c5ad3cc1935/app/models/enrollment.rb#L27)), there is no control of uniqueness on nextEnrollments. This could be error prone or induce weird behavior.

Reply:
Not sure I understand what is uniqueness control as I have no idea of the data schema ^^

Reply:
What I do understand is that if there are several next enrollments, all of them should appear on page